src/systems/hydrodynamics/Hydrodynamics.cc

```diff
@@ -350,12 +354,23 @@ void Hydrodynamics::Configure(
       prefix += "Dot";
       prefix += snameConventionVel[j];
       this->dataPtr->Ma(i, j) = SdfParamDouble(_sdf, prefix, 0);
+      addedMassSpecified &= (std::abs(this->dataPtr->Ma(i, j)) > 1e-6);
```

Comment:
Strictly speaking `&=` is equal to `<exp> & <exp>`, the bitwise operator, not the logical one. I wouldn't use it here, just to be safe.

Reply:
Thanks for spotting this! Fixed da9dc71bf.